CONTRIBUTING.md

```diff
@@ -74,38 +64,66 @@ The following guidelines should be observed when creating a PR.
-   be consistent with the `bbox` and `timestamp` of the `EOPatch`.
+* Your contribution should include unit tests, to test correct behaviour of the new feature and to lower the maintenance effort. Bug fixes as well as new features should include unit tests. When submitting the PR, check whether the Travis CI testing returns any errors, and if it does, please try to fix the issues causing failure. A test `EOPatch` is made available [here][test-eo-patch] with data for each `FeatureType`. Unit tests evaluating the correctness of new tasks should use data available in this `EOPatch`. New fields useful for testing purposes can be added, but should be consistent with the `bbox` and `timestamp` of the `EOPatch`. To execute all unit tests locally on you machine, use the command: `pytest` from the main folder. See also the [examples/README.md](examples/README.md) for how to setup you SentinelHub account and local config for testing.
 
- * Try to keep contributions small, as this speeds up the reviewing process. In the case of large
-   contributions, e.g. a new complex `EOTask`, it's best to contact us first to review the scope
-   of the contribution.
+* Try to keep contributions small, as this speeds up the reviewing process. In the case of large contributions, e.g. a new complex `EOTask`, it's best to contact us first to review the scope of the contribution.
 
- * Keep API compatibility in mind, in particular when contributing a new `EOTask`. In general,
-   all new tasks should adhere to the modularity of **eo-learn**.
-   Check the Section below for more information on how to contribute an `EOTask`.
+* Keep API compatibility in mind, in particular when contributing a new `EOTask`. In general, all new tasks should adhere to the modularity of **eo-learn**. Check the Section below for more information on how to contribute an `EOTask`.
 
- * New features or tasks should be appropriately commented using Sphinx style docstrings. The documentation uses
-   the [PEP-8][pep-8] formatting guidelines. [Pylint][pylint] is used to check the coding standard.
-   Therefore please run `pylint` from the the main folder, which contains the `pylintrc` file, to make sure your
-   contribution is scored `10.0/10.0`.
+* New features or tasks should be appropriately commented using Sphinx style docstrings. The documentation uses the [PEP-8][pep-8] formatting guidelines. [Pylint][pylint] is used to check the coding standard. Therefore, please run `pylint */ *.py` from the the main folder, which contains the `pylintrc` file, to make sure your contribution is scored `10.0/10.0`.
+
+### Development environment
+
+* Get the latest development version by creating a fork and clone the repo:
+
+```bash
+git clone git@github.com:<username>/eo-learn.git
+```
+
+* Make sure that you have python 3 installed, i.e. version 3.5 or higher. See https://www.python.org/downloads/ for general Python versions or https://www.anaconda.com/distribution/ for using Conda.
```

Comment:
Suggested changes:

Make sure that you have Python ... See [Python downloads][python] for general Python versions or [Anaconda distributions][conda] for using Conda.

and adding at the bottom to the rest of the links
[python]: https://www.python.org/downloads/
[conda]: https://www.anaconda.com/distribution/